### PR TITLE
Add the client name and ipport to the slowlog output

### PR DESCRIPTION
**Pull request by LiuYuHui**
Related to #1736

### CODE REVIEW

File: src/server/server.cc [hunk 1]
```diff
@@ -1407,7 +1407,8 @@ time_t Server::GetLastScanTime(const std::string &ns) {
   return 0;
 }
 
-void Server::SlowlogPushEntryIfNeeded(const std::vector<std::string> *args, uint64_t duration) {
+void Server::SlowlogPushEntryIfNeeded(const std::vector<std::string> *args, uint64_t duration, std::string client_name,
```
Comment (git-hulk) [inline src/server/server.cc]: Is it better to pass the `const Connection *conn` here? instead of passing its member one by one.

Comment (LiuYuHui) [reply src/server/server.cc]: Thanks, changed it.

**Review by torwig — DISMISSED** [review]
LGTM

File: src/server/server.cc [hunk 2]
```diff
@@ -1428,6 +1428,9 @@ void Server::SlowlogPushEntryIfNeeded(const std::vector<std::string> *args, uint
   }
 
   entry->duration = duration;
+  entry->client_name = std::move(conn->GetName());
+  entry->ip = std::move(conn->GetIP());
```
Comment (PragmaTwice) [inline src/server/server.cc]:
```suggestion
  entry->client_name = conn->GetName();
  entry->ip = conn->GetIP();
```

**Comment by PragmaTwice** [comment]
Thanks for your contribution!

Please refer to https://kvrocks.apache.org/community/contributing/ to format, lint and test your code. It is hard for us to merge it until the CI passes.

BTW, it would be better to add golang test cases for slowlog changes.

File: src/server/server.cc (hunk 2 above)
Comment (LiuYuHui) [reply src/server/server.cc]: thanks, fixed.

File: tests/gocase/unit/slowlog/slowlog_test.go [hunk 3]
```diff
@@ -161,4 +161,15 @@ func TestSlowlog(t *testing.T) {
...
 	})
 
+	t.Run("SLOWLOG - slowlog get output client name and ipport", func(t *testing.T) {
+		require.NoError(t, rdb.ConfigSet(ctx, "slowlog-log-slower-than", "0").Err())
+		require.NoError(t, rdb.Do(ctx, "client", "setname", "foobar").Err())
+		require.NoError(t, rdb.SAdd(ctx, "set", "foo", "bar").Err())
+		
+		val, err := rdb.SlowLogGet(ctx, -1).Result()
+		require.NoError(t, err)
+		require.EqualValues(t, "foobar" , val[0].ClientName)
```
Comment (git-hulk) [inline tests/gocase/unit/slowlog/slowlog_test.go]:
```suggestion
		require.EqualValues(t, "foobar", val[0].ClientName)
```

**Comment by git-hulk** [comment]
Others are good to me, thank you!

**Review by git-hulk — APPROVED** [review]

**Review by torwig — APPROVED** [review]

**Review by PragmaTwice — APPROVED** [review]

**Pull request merged**